jena-fuseki2/jena-fuseki-ui/src/services/mock/json-server.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const data = require('./json/index')
+
+const jsonServer = require('json-server')
+
+const server = jsonServer.create()
+const router = jsonServer.router(data)
+const middlewares = jsonServer.defaults()
+
+server.use(middlewares)
+
+// Have all URLS prefixed with /$
+server.use(
+  jsonServer.rewriter({
+    '/$/*': '/$1',
+    '/stats/ds1': '/ds1'
+  })
+)
```

Comment:
This `json-server` uses the JSON Server library to serve... err, well, JSON data. I used it in another project with GraphQL, and removes the need to have to start Jena and create a dataset and populate it in order for the tests to run (Skosmos does that, for example).